### PR TITLE
Cut v0.7.0

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
Releasing the first public version of Merlin, **v0.7.0**.

This PR contains the following changes:

* Add changelog file
* Update versioning across the project from 0.6.0* to 0.7.0
* Update Swagger spec and its API generated code
* Update documentations

### CODE REVIEW

**Comment by pradithya** [comment]
Can you change the PR title to a more descriptive name and also include the description.

**Comment by ariefrahmansyah** [comment]
> Can you change the PR title to a more descriptive name and also include the description.

Updated @pradithya

File: python/sdk/client/api/__init__.py [hunk 1]
```diff
@@ -1,17 +1,3 @@
-# Copyright 2020 The Merlin Authors
```
Comment (pradithya) [inline python/sdk/client/api/__init__.py:1]: Is it intended that all copyright header are removed?

Comment (ariefrahmansyah) [reply python/sdk/client/api/__init__.py:1]: For Python generated API code, yes. Previously the copyright header is not generated by Swagger Codegen

**Review by pradithya — APPROVED** [review]
LGTM

**Pull request merged**